matRad_calcParticleDose.m

```diff
@@ -186,471 +200,530 @@
-            for rangeShiftScen = 1:pln.multScen.totNumRangeScen
-               
-               if pln.multScen.scenMask(ctScen,shiftScen,rangeShiftScen)
-                  dij.mLETDose{ctScen,shiftScen,rangeShiftScen} = spalloc(dij.doseGrid.numOfVoxels,numOfColumnsDij,1);
-               end
-               
+        isfield(pln.propDoseCalc,'calcLET') && ...
+        pln.propDoseCalc.calcLET
+    if isfield(machine.data,'LET')
+        
+        letDoseTmpContainer = cell(numOfBixelsContainer,pln.multScen.numOfCtScen,pln.multScen.totNumShiftScen,pln.multScen.totNumRangeScen);
+        
+        for ctScen = 1:pln.multScen.numOfCtScen
+            for shiftScen = 1:pln.multScen.totNumShiftScen
+                for rangeShiftScen = 1:pln.multScen.totNumRangeScen
+                    
+                    if pln.multScen.scenMask(ctScen,shiftScen,rangeShiftScen)
+                        dij.mLETDose{ctScen,shiftScen,rangeShiftScen} = spalloc(dij.doseGrid.numOfVoxels,numOfColumnsDij,1);
+                    end
+                    
+                end
             end
-         end
-      end
-      
-   else
-      matRad_dispToConsole('LET not available in the machine data. LET will not be calculated.',param,'warning');
-   end
+        end
+        
+    else
+        matRad_dispToConsole('LET not available in the machine data. LET will not be calculated.',param,'warning');
+    end
 end
 
-
-% book keeping - this is necessary since pln is not used in optimization or
-% matRad_calcCubes
 if strcmp(pln.bioParam.model,'constRBE')
-   dij.RBE = pln.bioParam.RBE;
+    dij.RBE = pln.bioParam.RBE;
 end
 
-% ser overlap prioriites
+% set overlap prioriites
 cst = matRad_setOverlapPriorities(cst);
 
 % resizing cst to dose cube resolution
 cst = matRad_resizeCstToGrid(cst,dij.ctGrid.x,dij.ctGrid.y,dij.ctGrid.z,...
-   dij.doseGrid.x,dij.doseGrid.y,dij.doseGrid.z);
+    dij.doseGrid.x,dij.doseGrid.y,dij.doseGrid.z);
 
+vTissueIndex = zeros(size(VdoseGrid,1),1);
 % generates tissue class matrix for biological treatment planning and alpha_x, beta_x, vectors
 if pln.bioParam.bioOpt
-   
-   vTissueIndex = zeros(size(VdoseGrid,1),1);
-   dij.ax       = zeros(dij.doseGrid.numOfVoxels,1);
-   dij.bx       = zeros(dij.doseGrid.numOfVoxels,1);
-   dij.abx      = zeros(dij.doseGrid.numOfVoxels,1);  % alpha beta ratio
-   
-   % retrieve photon LQM parameter for the current dose grid voxels
-   [dij.ax,dij.bx] = matRad_getPhotonLQMParameters(cst,dij.doseGrid.numOfVoxels,1,VdoseGrid);
-   
-   dij.abx(dij.bx>0) = dij.ax(dij.bx>0)./dij.bx(dij.bx>0);
-   
-   % only if LEM is used corresponding bio data must be available in the base data set
-   if strcmp(pln.bioParam.model,'LEM')
-      if isfield(machine.data,'alphaX') && isfield(machine.data,'betaX')
-         
-         fprintf('matRad: loading biological base data... ');
-         
-         for i = 1:size(cst,1)
-            
+    
+    dij.alphaX  = zeros(dij.doseGrid.numOfVoxels,1);
```

Comment:
You like alphaX and betaX more that ax and bx?

Reply:
You started this in your APM branch and I thought it's clearer to read.